src/psyclone/tests/psyir/nodes/scoping_node_test.py

```diff
@@ -231,7 +231,8 @@ def test_scoping_node_equality():
     symboltable = SymbolTable()
     symboltable2 = SymbolTable()
     sched1 = Schedule(symbol_table=symboltable)
-    sched2 = Schedule(symbol_table=symboltable)
+    sched2 = Schedule()
```

Comment:
Please add a comment here and update the copyright date.

Reply:
Done